src/concurrency/genmc/run.rs

```diff
@@ -20,16 +20,19 @@ pub fn run_genmc_mode<'tcx>(
     let genmc_ctx = Rc::new(GenmcCtx::new(config, target_usize_max));
 
     for rep in 0u64.. {
+        // will panic on wrap-around
```

Comment:
This comment was meant to go on the previous line. I guess rustfmt move it... please put it above the loop then and make it clear that it refers to the `0..` iterator.